external/js/cothority/spec/byzcoin/stream.spec.ts

```diff
@@ -23,16 +22,15 @@ describe("Stream Tests", () => {
 
         const msg = new PaginateRequest({startid: rpc.genesisID, pagesize: 1, numpages: 1,  backward: false});
 
-        const foo = {
-            onClose: (code: number, reason: string) => {
-                expect(code).toEqual(1000);
```

Comment:
Ok so we loose access to the close code and reason. Hope we won't need it.

Reply:
if the `code` and `reason` of the close is something that points to a non-error, it will resolve to `error`. So we have to make sure that the test for 'non-error' is correct.

Reply:
"if (...) that points to a non-error, it will resolve to `error`". 

why?

Reply:
https://github.com/dedis/cothority/pull/2211/files#diff-5f68f2cd364b6d32e0a39977416c8cdeR189